### DISCUSSION TITLE
Feature Request: better docs for reading parquets in S3

### DISCUSSION THREAD

**Discussion by raybellwaves** [duckdb/duckdb#14498] [Ideas]
As discussed in https://discord.com/channels/909674491309850675/921100786098901042/1294282985788412005

I have to authenticate using

```
import boto3
aws_creds = boto3.Session().get_credentials()
conn = duckdb.connect()
conn.install_extension("httpfs")
conn.install_extension("aws")
conn.load_extension("httpfs")
conn.load_extension("aws")
conn.execute(
f"""
CREATE SECRET s3_secret (
    TYPE S3,
    KEY_ID '{aws_creds.access_key}',
    SECRET '{aws_creds.secret_key}',
    SESSION_TOKEN '{aws_creds.token}'
);
SELECT * FROM read_parquet('s3://BUCKET/FILE.parquet');
"""
)
```

I'd like to make a PR to document this in the duckdb docs

**Comment by soerenwolfers** [comment]
![1000024999](https://github.com/user-attachments/assets/daa4a4b2-3396-4f45-90b2-b9e3c3fd9802)

**Reply by raybellwaves** [reply]
https://duckdb.org/docs/api/python/data_ingestion may be a better place for it as it uses boto3 as opposed to https://duckdb.org/docs/extensions/httpfs/s3api.html ?
